.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+<table>
+    <thead>
+        <tr>
+            <th>Title</th>
+            <th>Description</th>
+            <th>Labels</th>
+            <th>Priority</th>
+            <th>Assignees</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=6>A high-level summary of the request</td>
+            <td>Requirement: why is this feature needed? </td>
+            <td rowspan=6>Add labels to indicate what the task is about. A label can also be used to flag tasks for refinement. </td>
+            <td rowspan=6>Low, medium, high, blocker</td>
+            <td rowspan=6>Determine assignees before the upcoming sprint</td>
+        </tr>
+        <tr>
+            <td>Proposed solution(s): describe technical solutions</td>
```

Comment:
I have one question/suggestion/doubt - feature requests can be made not only (in our case, and generally even mainly) by non-technical people, so suggesting technical solutions might be an issue. Also I think that it should be on dev's side to propose a technical solution for the requested feature. What does the team think?

Reply:
Agree. Also, it can be rephrased to "Proposed solution(s)"  and that's it. Because maybe people who are questioning the feature know what has to be done, but not in tech. way. And we, as developers, during refinement can add more details